homework_12/myproject/homework_model/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+from django.db.models import TextField
+
+# Create your models here.
+
+
+class Person(models.Model):
```

Comment:
in this way, table for Person would also be created. We actually do not need this (it leads to redundant relations, see migrations file). try https://docs.djangoproject.com/en/3.2/topics/db/models/#abstract-base-classes